examples/applications/plot_out_of_core_classification.py

```diff
@@ -192,12 +201,16 @@ def progress(blocknum, bs, size):
 # Create the vectorizer and limit the number of features to a reasonable
 # maximum
 
-vectorizer = HashingVectorizer(decode_error='ignore', n_features=2 ** 18,
-                               alternate_sign=False)
 
+vectorizer = HashingVectorizer(
+    decode_error='ignore', n_features=2 ** 18, alternate_sign=False)
 
 # Iterator over parsed Reuters SGML files.
-data_stream = stream_reuters_documents()
+# Train Datasets
+data_stream_train = stream_reuters_documents(train_test="TRAIN")
+
+# Test Datasets
+data_stream_test = stream_reuters_documents(train_test="TEST")
```

Comment:
I've now realised that we are doing two passes through the steam which kind of defeats the purpose. Either the test set is a prefix or we collect it while passing it through the stream

Reply:
No comment?